src/Propel/Generator/Manager/AbstractManager.php

```diff
@@ -364,6 +364,12 @@ protected function includeExternalSchemas(\DOMDocument $dom, $srcDir)
 
             $externalSchema->parentNode->removeChild($externalSchema);
 
+            $relativePath = realpath($srcDir. '/'. $include);
+
+            if (!file_exists($include) && file_exists($relativePath)) {
```

Comment:
`!file_exists($include)` should be removed tho

Reply:
Can I use is_readable function in this condition?

Reply:
Or do you think I should remove the first part of the condition?

Reply:
`if (file_exists($relativePath)) {` should be enough